### DISCUSSION TITLE
Postgres migrate error when upgrading from 3.1.11 to 3.2.0

### DISCUSSION THREAD

**Discussion by sml12179** [netbox-community/netbox#9093] [Q&A] [answered]
I am still having issues upgrading from 3.1.11 to 3.2.0. I am using netbox-docker. I do not have any plugins, I have tried manage.py migrate this did not resolve the issue. I can start with a fresh install of 3.1.11 and have no errors when loading the db from db_dump. Every time I follow the procedures outlined on the netbox-docker wiki I get the following. Also I had no ASNs or contacts assigned to site but ran the migration script anyways and it reported that there were none. 
[nb3110_320 error.txt](https://github.com/netbox-community/netbox/files/8457451/nb3110_320.error.txt)

**Comment by sml12179** [comment, marked as answer]
I was able to resolve this.

**Reply by tyler-8** [reply]
This feels very much like this comic https://xkcd.com/979/
